notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
```

Comment:
@ambrosejcarr 
A lot of the plotting code in here will be useful for QC on all multiplexed assays. I fully intend to rip out these plotting functions and add them to a starfish.quality or a starfish.stats or something like that. But for now, I've decided to leave this code in the notebook and will address the changes in a future PR.